Committees/National_Political_Education_Committee.adoc

```diff
@@ -0,0 +1,29 @@
+= National Political Education Committee
+
+== Purpose
+
+Political education of PLTCom members is the sole mission of this committee. It will provide all members a course in capitalism first and scientific socialism second followed by a course on party work. It will also maintain a comprehensive digital library of marxist political material as well as a physical library as possible. 
+
+== Structure
+
+Only _full_ members can hold these positions:
+
+- Committee Chair
+- Librarian
+- Teacher(s)
+
+== Meetings
+
+This committee will meet at a minimum of 4 times per month to discuss and plan educational curriculum. 
+
+== Reports
+
+This committee will submit a report to the Central Committee every other meeting detailing: work hours, meetings held, students, and a summary of events.

```

Comment:
We should coordinate on a standard report

Reply:
Should we require committees report hours contributed, tasks set, tasks completed, and summary of activity for the period?

Reply:
Right now, the survey asks for a summary of the progress made, if the committee is on track with the goals set in the one year plan, and if they are experiencing issues and gives the description of a few categories of issues: Monetary, Material, Personnel, or Engagement. If the committee reports issues, it asks for a description, if they have a plan, and a short description of the plan (if they have one.)

At the end of the survey, it asks everyone regardless of progress if there's anything leadership can do to help them. 

What is the purpose of reporting hours contributed? 

Tasks set and completed could be added, and appended to the one year plan as they complete the tasks they've already set out for themselves.

Reply:
I think keeping track of hours spent would give a better indication of where most effort is being targeted and if more support should be allocated.